notebooks/GWAS_1000_genomes_p2/main.py

```diff
@@ -0,0 +1,844 @@
+    2. the phenotype filepath in the workspace storage (this should start with *gs://*)
+    3. the column name in the phenotype file that corresponds to the sample IDs (if you followed the hint above, this should be *sample_id*)
+    4. the outcome you would like to model (this must be a column name in the phenotype file)
+    5. a list of covariates to include in the model (these should be column names in the phenotype file separated by commons)
+
+    You can <font color='red'>makeSampleSet</font> with the following syntax. If you'd like to track different covariates and outcomes, you can change those values
+    """
+
+with herzog.Cell("python"):
+    label = "tutorial-analysis-vcfupdate"
+    phenotype_file = bucket + "my_phenotypes.csv"
+    sample_id_column = "subject_id"
+    outcome = "bmi"
+    covariates = "age, sex, population"
+    grm = bucket + "kinship.csv"
+    makeSampleSet(samples, PROJECT, WORKSPACE, label, phenotype_file, sample_id_column, outcome, covariates)
+
+with herzog.Cell("python"):
+    vcf_attribute = [fiss.fapi._attr_set("vcfs", vcf_filtered_array)]
+    fiss.fapi.update_entity(PROJECT, WORKSPACE, 'sample_set', 'tutorial-analysis-vcfupdate', vcf_attribute)
+
+with herzog.Cell("markdown"):
+    """
+    # Next steps
+    Congratulations on completing the first steps in your GWAS analysis! You've loaded and explored the phenotype data to understand how the data are distributed, loaded the VCF data and performed variant QC, filtered variants and done LD-pruning to filer the matrix table, run a PCA, and added PCA values to the matrix table. You've saved the sample data and updated it to your workspace data table.
+    The next step is to use the data to do a mixed-model association test in a workflow. Save and close this notebook and proceed to the dashboard for instructions of how to do this.
+
+    ## Citations:
+        1. A global reference for human genetic variation, The 1000 Genomes Project Consortium, Nature 526, 68-74 (01 October 2015) doi:10.1038/nature15393
+        2. Yengo L, Sidorenko J, Kemper KE, Zheng Z, Wood AR, Weedon MN, Frayling TM, Hirschhorn J, Yang J, Visscher PM, GIANT Consortium. (2018). Meta-analysis of genome-wide association studies for height and body mass index in ~700, 000 individuals of European ancestry. Biorxiv
+        3. Scott RA et al. Large-scale association analyses identify new loci influencing glycemic traits and provide insight into the underlying biological pathways. Nat. Genet. 2012;44;9;991-1005
+        4. Klarin D, et al. Genetics of blood lipids among ~300, 000 multi-ethnic participants of the Million Veteran Program. Nat. Genet. 2018;50:1514–1523. doi: 10.1038/s41588-018-0222-9.
+        5. Hail Team. Hail 0.2.13-81ab564db2b4. https://github.com/hail-is/hail/releases/tag/0.2.13.
+
+    ## Authorship
+    These notebooks were converted by Beth Sheets (UCSC) and Ash O'Farrell (UCSC) as part of the Biodata Catalyst grant.
+
```

Comment:
Much of the source code and tutorial was developed by Tim Majarian in the Manning Lab at the Broad Institute. The notebooks were developed to work in the BioData Catalyst ecosystem by Beth Sheets (UCSC) and Ash O'Farrell (UCSC) as part of the NHLBI BioData Catalyst grant.

Reply:
were not was